core/src/main/scala/org/ensime/indexer/orientdb/SimpleOrient.scala

```diff
@@ -372,7 +373,7 @@ package object syntax {
       bdf: BigDataFormat[FqnSymbol],
       oid: OrientIdFormat[FqnSymbol, P],
       p: SPrimitive[P]
-    ): Seq[VertexT[FqnSymbol]] = {
+    ): Seq[(VertexT[FqnSymbol], VertexT[LineNumber])] = {
```

Comment:
I don't like that this returns a Seq of Tuples. It should return a `Seq` of some domain object.